pkgs/tools/networking/boundary/default.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv, lib, fetchurl, unzip }:
+
+stdenv.mkDerivation rec {
+  pname = "boundary";
+  version = "0.1.4";
+
+  src = let
+    inherit (stdenv.hostPlatform) system;
+    suffix = {
+      x86_64-linux = "linux_amd64";
+      aarch64-linux = "linux_arm64";
+      x86_64-darwin = "darwin_amd64";
+    }."${system}" or (throw "Unsupported system: ${system}");
+    fetchsrc = sha256: fetchurl {
```

Comment:
I think you can use fetchzip here and remove the unzip package and unpackPhase.

Reply:
I just tried `fetchzip` and `nix-prefetch-url --unpack` and the SHAs dont match. Updating the package will require me to build on all platforms manually and copy out the SHAs which isn't worth it. Once resolved I'll swap it to `fetchzip`

Related issue:
#111508

Reply:
Switching to https://github.com/msteen/nix-prefetch allowed me to use `fetchzip`
https://github.com/NixOS/nix/issues/4499#issuecomment-770833444